pandas/core/frame.py

```diff
@@ -6921,6 +6921,11 @@ def unstack(self, level=-1, fill_value=None):
         Name to use for the 'value' column.
     col_level : int or str, optional
         If columns are a MultiIndex then use this level to melt.
+    ignore_index : bool, optional, default True
+        If False, original index is ignored. If True, the original index is retained.
```

Comment:
is this correct? if ignore_index=False, the index is ignored?

Reply:
Oh my word... Thanks for being so thorough!